dbhydra/src/tables.py

```diff
@@ -880,28 +878,30 @@ def modify_column(self, column_name, new_column_type):
 ############### XLSX ##################
 
 class XlsxTable(AbstractTable):
-    def __init__(self, db1, name, columns=None, types=None, id_column_name = "id"):
+    def __init__(self, db1, name, columns=None, types=None, id_column_name="id", use_csv=False):
         super().__init__(db1, name, columns)
         self.types = types
-        self.id_column_name=id_column_name
-        self.table_directory_path: pathlib.Path = self.db1.db_directory_path / f"{self.name}.xlsx"
+        self.id_column_name = id_column_name
+        self.use_csv = use_csv
+
+        table_filename = f"{self.name}.csv" if self.use_csv else f"{self.name}.xlsx"
+        self.table_directory_path: pathlib.Path = self.db1.db_directory_path / table_filename
+
+    def _save_table(self, df: pd.DataFrame):
+        if self.use_csv:
+            df.to_csv(self.table_directory_path, index=False)
+        else:
+            df.to_excel(self.table_directory_path, index=False)
 
     def create(self):
         if not self.table_directory_path.exists():
-            df=pd.DataFrame(columns=self.columns)
-            df.to_excel(self.table_directory_path, index=False)
+            df = pd.DataFrame(columns=self.columns)
+            self._save_table(df)
         else:
             print(f"Table '{self.name}' already exists")
 
     def drop(self):
-        if self.table_directory_path.exists():
-            if sys.platform in ["darwin","linux","linux2"]:
-                command="rm "+str(self.table_directory_path)
-            else:
-                command="del "+str(self.table_directory_path)
-            os.system(command)
-        
-
+        self.table_directory_path.unlink(missing_ok=True)
```

Comment:
how does this work?

Reply:
should have comment

Reply:
Are you sure its equivalent to drop the folder structure?

Reply:
This is not function for removing a folder, but rather for removing a file (see `self.table_directory_path` definition, the name though is misleading and could be rather `self.table_path`, but I just kept the original one). Anyway `unlink` method will remove file.

Reply:
And overall calling os-specific commands directly is not the best practice, especially when there is a dedicated library for that :)

Reply:
Ah you are right - it's confusing name - it shouldnt be named "table_directory_path"